pipeline/__init__.py

```diff
@@ -0,0 +1,330 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
the "pipeline" module name needs to be changed to something better that will not conflict when installed with other libraries.

I suggest either:

- using a namespace (we never did before, but we could start). This would mean creating a top level `aboutcode` directory with no `__init__.py` and a few more details TBD. This is PEP420 https://peps.python.org/pep-0420/ and https://docs.python.org/3/glossary.html#term-namespace-package  and this namespace would become reusable across other projects for shared modules. This is OK because we own aboutcode on PyPI and none of our projects ever used the `aboutcode` module name. Some exmaples include jaraco's repos at https://github.com/jaraco/jaraco.vcs https://github.com/jaraco/jaraco.abode and https://github.com/jaraco/jaraco.context all living in the same "jaraco" namespace.
In this case I would suggest that we call the package pipeline under aboutcode namespace. And end up with this path `aboutcode/pipeline/__init__.py`

- using another name for the package directory that contains `__init__.py` and use that as a regular, non-namespaced module. For instance `aboutcode_pipeline/__init__.py`
 
There are no specific benefits nor issues with the second approach. Using the first approach with a namespace would be the pythonic way.

We should  name the wheel `aboutcode.pipeline` to make it unique.